crates/ol/state-types/Cargo.toml

```diff
@@ -11,11 +11,12 @@ strata-codec-utils.workspace = true
 strata-identifiers.workspace = true
 strata-ledger-types.workspace = true
 strata-merkle.workspace = true
+strata-predicate.workspace = true
 strata-snark-acct-types.workspace = true
 
 bitcoin.workspace = true
 ssz.workspace = true
-tree_hash.workspace = true
+ssz_derive.workspace = true
```

Comment:
move `ssz_*` and `tree_hash*` deps to the `strata-*` block. They are git repos internal to `alpenlabs` GitHub org.